mlrun/model_monitoring/db/tsdb/tdengine/tdengine_connector.py

```diff
@@ -58,7 +58,12 @@ def _create_connection(self):
         except taosws.QueryError:
             # Database already exists
             pass
-        conn.execute(f"USE {self.database}")
+        try:
+            conn.execute(f"USE {self.database}")
+        except taosws.QueryError as e:
+            raise mlrun.errors.MLRunInvalidArgumentError(
```

Comment:
Maybe use MLRunNotImplementedServerError?
Using MLRunInvalidArgumentError does not seem related here..

Reply:
amm `MLRunNotImplementedServerError` seems to be related to not supported features, what do you think about `MLRunInternalServerError`? because basically mlrun server is not able to connect these external services so maybe it does make sense.